src/en/charms-storage.md

```diff
@@ -237,6 +238,53 @@ default values:
 juju deploy postgresql --storage pgdata=ebs,10G cache=ebs-ssd
 ```
 
+### Remove storage
+
+If you remove a unit that has storage, the storage is removed with the unit.
+
+The long term plan is to enable storage to be managed separately from units
```

Comment:
I would suggest we rephrase this in terms of Juju. E.g. 

Future versions of Juju will manage storage entirely separately from units. This will enable types of storage which can persist (e.g. EBS volumes) to be re-used by new units or workloads. This is not fully implemented in the current version of Juju, but one of the first steps is to be able to remove storage, which _is_ currently possible.